domain-locker/docker-compose.yml

```
@@ -0,0 +1,32 @@
+version: '3.7'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: domain-locker_app_1
+      APP_PORT: 3000
+
+  app:
+    image: lissy93/domain-locker:0.0.6@sha256:bd8b494b2a8b02e9fb612859f64c336e6eb1b21745199253b130705bd4ac8853
+    user: "1000:1000"
+    restart: on-failure
+    environment:
+      DL_ENV_TYPE: ${DL_ENV_TYPE:-selfHosted}
+      DL_PG_HOST: ${DL_PG_HOST:-postgres}
+      DL_PG_PORT: ${DL_PG_PORT:-5432}
+      DL_PG_USER: ${DL_PG_USER:-postgres}
+      DL_PG_PASSWORD: ${DL_PG_PASSWORD:-changeme2420}

```

I think you can remove the variable names here as those will not be used anyways. And maybe change the value to something else besides `changeme`.
```suggestion
      DL_PG_PASSWORD: domain-locker

```